websites/M/MyAnimeList.Vip/presence.ts

```diff
@@ -0,0 +1,84 @@
+const presence = new Presence({
+    clientId: "861544419005169675"
+  }),
+  browsingStamp = Math.floor(Date.now() / 1000);
+
+let iframeData: {
+  currTime: number;
+  duration: number;
+  paused: boolean;
+} = null;
+
+presence.on(
+  "iFrameData",
+  (data: { currTime: number; duration: number; paused: boolean }) => {
+    if (!data.paused) iframeData = data;
+  }
+);
+
+presence.on("UpdateData", async () => {
+  const presenceData: PresenceData = {
+    largeImageKey: "logo",
+    startTimestamp: browsingStamp
+  },
+    { pathname } = document.location;
+
+  if (pathname === "/") {
+    if (document.location.search) {
+      presenceData.details = `Procurando por ${document.location.search.substring(
+        3
+      )}`;
+    } else presenceData.details = "Explorando MyAnimeList.vip";
```

Comment:
wait actually you can make it simplier
```suggestion
      presenceData.details = document.location.search ? `Procurando por ${document.location.search.substring(
        3
      )}` : "Explorando MyAnimeList.vip";
```